runtime/testing-basilisk/src/xcm.rs

```diff
@@ -0,0 +1,238 @@
+use super::*;
```

Comment:
can this be in common?

Reply:
we should try to reuse as much stuff as possible in common and separate it only when necessary

Reply:
I am afraid it will not be possible. Almost everything in the crate depends on items from the runtime crate or on various xcm-related crates.